synapse/handlers/auth.py

```diff
@@ -887,17 +903,18 @@ def delete_threepid(self, user_id, medium, address, id_server=None):
         yield self.store.user_delete_threepid(user_id, medium, address)
         return result
 
-    def _save_session(self, session):
+    def _save_session(self, session: dict) -> None:
+        """Update the time a session was last used and add it back to the session store."""
```

Comment:
Maybe mention what time is used as the updated time? Looking at the code it's obvious that it's the current time, but if you only look at the function's doc it can be now but it could also be e.g. some value stored in the dict.

Reply:
This could be clearer, maybe:

> Update the last used time on the session to now and add it back to the session store.